changelog.d/7468.bugfix

```diff
@@ -0,0 +1 @@
+Ensure complete snaps can be build.
```

Comment:
could you try to better describe the problem being fixed here? Was it previously possible to build *incomplete* snaps somehow?

Reply:
also, since this only affects the snap build, rather than fixing an actual bug in the application, I think this should probably be a `.misc` file?